### DISCUSSION TITLE
Nozzle way too far from the bed

### DISCUSSION THREAD

**Discussion by nakwada** [dz0ny/klipper-prusa-mk3s#57] [Q&A]
Hello,

I followed the instructions here and successfully prepared my MK3S to run with Klipper.

However, upon starting my first print (sliced with PrusaSlicer with my usual settings), the nozzle is way too far from the bed.

Yes, I adjusted z offset in the printer.cfg as per instructions.

I am new to Klipper and I'd appreciate guidance. Thanks :)

**Comment by confuzled-dev** [comment]
The probing logic is different so this is possible. Just adjust your new offset until it is good

**Reply by nakwada** [reply]
mmmh, I am missing something. I ran PROBE_CALIBRATE, but afterwards the TESTZ command is unknown.

**Comment by confuzled-dev** [comment]
I never ran testz on my prusa

**Reply by nakwada** [reply]
Somehow I did something and now it works. I believe running BED_MESH_CALIBRATE, followed by SAVE_CONFIG did the trick.
